src/services/FirestoreTypes.ts

```diff
@@ -1,8 +1,16 @@
 export const DEALS_TOKEN_SWAP_COLLECTION = "deals-token-swap";
 export const DEALS_TOKEN_SWAP_UPDATES_COLLECTION = "deals-token-swap-updates";
 export const FIREBASE_MESSAGE_TO_SIGN = "Authenticate access to Prime Deals at";
+export const DEEP_DAO_COLLECTION = "deep-dao";
 
 export interface IFirebaseDocument<T = any> {
   id: string;
   data: T;
 }
+
+export interface IDAOsData
+{
```

Comment:
Please look over all the changes and put `{` on the same line and not separate line (because the code base has this code style)